### ISSUE TITLE
"An error occurred before the game window could be initialised. Please check the console!"

### ISSUE THREAD

**Issue by thewreck** [Davidobot/love.js#20]
Trying out the latest node.js with an empty/tiny/big/any löve2d project, I get this error:
`"An error occurred before the game window could be initialised. Please check the console!"`
, as soon as the `preparing` phase is done.

I have tried building the projects with every commit with changes to the `wasm` in this repo from:
```
Revision: 5a5f5f3b734affc6553d89e168d7095febabdd83
Date: 10/26/2020 2:28:18 PM
Focus window on loading and on clicking
```
to
```
Revision: a4eb3ee735ef70bbbee2964e361aee450a72c491
Date: 7/4/2017 7:29:57 PM
0.11.0-rc3
```
and all of them fail to load the game - however this specific error stopped after around 10 commits back from HEAD - at which point the "preparing game box" just disappears instead without errors and the page is just the theme, no canvas, no game.

I tried packaging the love.js `/spec/ ` project, and it fails in the same way.
Tried on **Chrome**, **Edge** and **Firefox** on **Windows 10**

The example projects linked (_Specification Test_, _Another Kind of World_ and _groverburger's 3D engine_) in the readme all work fine for me, except that the Specification Test sometimes randomly fails to load - hitting F5 has about a 50% success rate. They work both in standard and compatibility mode.

Have tried running it by both:
`node index.js` and `love-js`
and there does not seem to be any errors or issues building.

```
node version: v14.15.1
npm version: 6.14.8
```

Any idea what this could be? It feels like I'm missing something obvious.

**Comment by Davidobot** [comment]
Hm, what does the console print out? If you can, upload a love file for me to try out.

**Comment by thewreck** [comment]
After much testing - I think we found the problem:

We first tried 7zip to make the zip (using the regular .zip option) and tested that it worked standalone, and tried building with that - and it did not work. Then we went on to try using regular folders, and it also did not work, so we thought that it did not matter.

Now we tried in a desperate final attempt, to use regular windows `send to -> compress (zipped) folder` and THAT WORKS.

Thanks for your quick response, loving the project.
